src/v2/view-builder/views/email/EnrollAuthenticatorEmailView.js

```diff
@@ -1,16 +1,129 @@
 import { loc } from 'okta';
+import hbs from 'handlebars-inline-precompile';
+import { View } from 'okta';
 import BaseAuthenticatorEmailView from './BaseAuthenticatorEmailView';
 
 const BaseAuthenticatorEmailForm = BaseAuthenticatorEmailView.prototype.Body;
 
-const Body = BaseAuthenticatorEmailForm.extend(Object.assign({
-
-  resendEmailAction: 'currentAuthenticator-resend',
+const CheckYourEmailTitle = View.extend({
```

Comment:
I think all the logic/components being reused from `ChallengeAuthenticatorEmailView.js` can be extrapolated into a new file where both files can import from, let me know what you think?

Reply:
Yes some methods can be reused from `ChallengeAuthenticatorEmailView` and Ill move it to new file.

Reply:
Done